releasenotes/notes/profiling-option-to-disable-endpoint-collection-48ab251acb0d9b4e.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add support for `DD_PROFILING_ENDPOINT_COLLECTION_ENABLED` env vaireble
```

```suggestion
    Add support for `DD_PROFILING_ENDPOINT_COLLECTION_ENABLED` env variable
```

---

```suggestion
    Add support for ``DD_PROFILING_ENDPOINT_COLLECTION_ENABLED`` env variable
```

I missed it, also needs double <code>``</code>` for inline code blocks.